### PR TITLE
fix: address doctor validation

### PR DESCRIPTION
**Pull request by kreed0422**
### Please check if the PR fulfills these requirements:

- [ ] After these changes, the app was run and still works as expected
- [ ] Unit tests for these changes were added (if applicable)
- [ ] All existing unit tests were run and still pass

### CODE REVIEW

File: fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h2 class="border-bottom">
         <div class="form-group">
           <ng-container  *ngIf="isAddressValidatorEnabled; else UseGeoCoder">
           <common-address-validator label='(optional) Address Lookup'
-                                    [(address)]='applicant.ls'
+                                    [(address)]='applicant.updAddress'
```
Comment (harrymaynard-maximus) [inline fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.html:37]: Was this an intentional change?

File: fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.ts [hunk 2]
```diff
@@ -147,6 +146,8 @@ export class MailingAddressPageComponent extends AbstractFormComponent implement
   }
 
   onAddressSelected(address: Address) {
+    console.log( 'onAddressSelected: ', address );
+
```
Comment (harrymaynard-maximus) [inline fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.ts:150]: Please remove console log.

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

File: fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.html (hunk 1 above)
Comment (kreed0422) [reply fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.html:37]: Typo from a previous commit

File: fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.ts (hunk 2 above)
Comment (kreed0422) [reply fpcare/src/app/modules/registration/pages/mailing-address/mailing-address.component.ts:150]: removed console log

**Review by harrymaynard-maximus — APPROVED** [review]

**Pull request merged**